apps/nightscout/nightscout.star

```diff
@@ -186,21 +202,45 @@ def main(config):
                 cross_align="center",
                 expanded=True,
                 children=[
-                    render.Text(
-                        content=str(int(sgv_current)),
-                        font="6x13",
-                        color=font_color
+                    render.Column(
+                        cross_align = "center",
+                        main_align = "space_evenly",
+                        expanded = True,
+                        children = [
+                            render.Text(
+                                content=str(int(sgv_current)),
+                                font="6x13",
+                                color=font_color
+                            ),
+                            render.Text(
+                                content="Updated:",
+                                font="tom-thumb"
+                            ),
+                            render.Marquee(
+                                width = 32,
+                                child=render.Text(
+                                    # This time is currently in UTC - need to figure out how to display in a selected timezone.
+                                    content=latest_reading_dt.format("2006-01-02 03:04:05 PM"),
```

Comment:
Still todo: add location to the scheme and convert UTC to the appropriate timezone